src/main/java/com/vdmytriv/bookstoreapp/model/Book.java

```diff
@@ -37,4 +39,7 @@ public class Book {
     private String description;
 
     private String coverImage;
+
+    @Column(nullable = false)
+    private boolean deleted = false;
```

Comment:
boolean variables usually start from "is..."
```suggestion
    private boolean isDeleted = false;
```